src/Facebook/InstantArticles/Client/Client.php

```diff
@@ -58,6 +58,7 @@ public function __construct($facebook, $pageID, $developmentMode = false)
      * @param string $pageID Specify the Facebook Page to use for Instant Articles
      * @param bool $developmentMode|false Configure the service to use the Instant Articles development sandbox
      *
+     * @return static
```

Comment:
Guess this is more <type> <what is inside> standard.
Can you add this, please?
`@return Client new Instance.`